UPGRADING

```diff
@@ -33,6 +33,10 @@ PHP 7.4 UPGRADE NOTES
 2. New Features
 ========================================
 
+- Core:
+  . The array_key_exists() function is now optimized into a specialized
+    ZEND_ARRAY_KEY_EXISTS instruction.
+
```

Comment:
This doesn't change anything for regular users. I would move this into UPGRADING.INTERNALS or completely remove.

Reply:
Alright. :ok_hand:

Reply:
I'd say let's leave the decision up to the 7.4 RM?